ldms/src/test/Makefile.am

```diff
@@ -39,4 +39,29 @@ test_metric_SOURCES = test_metric.c
 test_metric_LDADD = -lldms
 test_metric_LDFLAGS = $(AM_LDFLAGS) -pthread -lm
 
-TESTS = $(check_PROGRAMS)
+EXTRA_DIST=ldms-run-static-tests.sh.in
+EXTRA_DIST+=ldms-static-test-list.sh.in
+EXTRA_DIST+=ldms-static-test-bypass.in
+
+CLEANFILES=ldms-run-static-tests.sh
+CLEANFILES+=ldms-static-test-list.sh
+CLEANFILES+=ldms-static-test-bypass 
+
+%.sh: %.sh.in
```

Comment:
This is a sign that something is wrong. These files should be generated at configure time because you listed them in AC_CONFIG_FILES. This should not be here.